src/compiler/CompileContext.js

```diff
@@ -137,6 +137,7 @@ class CompileContext extends EventEmitter {
         this.compilerVersion = this.options.compilerVersion || markoPkgVersion;
         this.writeVersionComment = writeVersionComment !== 'undefined' ? writeVersionComment : true;
         this.ignoreUnrecognizedTags = this.options.ignoreUnrecognizedTags || false;
+        this.escapeAtTags = this.options.escapeAtTags || false;
```

Comment:
Is this used anywhere?  Did you intend to use it in the `if` statements below?  Or is it just to have easy access from elsewhere?